config/peerpods/podvm/Dockerfile.podvm-builder.openshift

```diff
@@ -0,0 +1,40 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.5-1736404036
+
+# azure-podvm-image-handler.sh script under /scripts/azure-podvm-image-handler.sh
+# aws-podvm-image-handler.sh script under /scripts/aws-podvm-image-handler.sh
+# sources for cloud-api-adaptor under /src/cloud-api-adaptor
+# The podvm binaries are expected to be under /payload/podvm-binaries.tar.gz
+# Binaries like kubectl, packer and yq under /usr/local/bin will be installed by the scripts
+
+
+LABEL kata_src=https://github.com/openshift/kata-containers
+LABEL kata_src_commit=osc-release
+
+ARG ORG_ID
+ARG ACTIVATION_KEY
+
+RUN mkdir -p /scripts
+
+ADD lib.sh libvirt-podvm-image-handler.sh aws-podvm-image-handler.sh azure-podvm-image-handler.sh /scripts/
```

Comment:
Probably this will conflict with #519 but I can handle on my side. Just adding a note so we (I) don't  forget :)